src/fseccomp/syscall.c

```diff
@@ -190,6 +283,382 @@ static const SyscallGroupList sysgroups[] = {
+#ifdef SYS_getxattr
+	  "getxattr,"
+#endif
+#ifdef SYS_inotify_add_watch
+	  "inotify_add_watch,"
+#endif
+#ifdef SYS_inotify_init
+	  "inotify_init,"
+#endif
+#ifdef SYS_inotify_init1
+	  "inotify_init1,"
+#endif
+#ifdef SYS_inotify_rm_watch
+	  "inotify_rm_watch,"
+#endif
+#ifdef SYS_lgetxattr
+	  "lgetxattr,"
+#endif
+#ifdef SYS_link
+	  "link,"
+#endif
+#ifdef SYS_linkat
+	  "linkat,"
+#endif
+#ifdef SYS_listxattr
+	  "listxattr,"
+#endif
+#ifdef SYS_llistxattr
+	  "llistxattr,"
+#endif
+#ifdef SYS_lremovexattr
+	  "lremovexattr,"
+#endif
+#ifdef SYS_lsetxattr
+	  "lsetxattr,"
+#endif
+#ifdef SYS_lstat
+	  "lstat,"
+#endif
+#ifdef SYS_lstat64
+	  "lstat64,"
+#endif
+#ifdef SYS_mkdir
+	  "mkdir,"
+#endif
+#ifdef SYS_mkdirat
+	  "mkdirat,"
+#endif
+#ifdef SYS_mknod
+	  "mknod,"
+#endif
+#ifdef SYS_mknodat
+	  "mknodat,"
+#endif
+#ifdef SYS_mmap
+	  "mmap,"
+#endif
+#ifdef SYS_mmap2
+	  "mmap2,"
+#endif
+#ifdef SYS_munmap
+	  "munmap,"
+#endif
+#ifdef SYS_newfstatat
+	  "newfstatat,"
+#endif
+#ifdef SYS_oldfstat
+	  "oldfstat,"
+#endif
+#ifdef SYS_oldlstat
+	  "oldlstat,"
+#endif
+#ifdef SYS_oldstat
+	  "oldstat,"
+#endif
+#ifdef SYS_open
+	  "open,"
+#endif
+#ifdef SYS_openat
+	  "openat,"
+#endif
+#ifdef SYS_readlink
+	  "readlink,"
+#endif
+#ifdef SYS_readlinkat
+	  "readlinkat,"
+#endif
+#ifdef SYS_removexattr
+	  "removexattr,"
+#endif
+#ifdef SYS_rename
+	  "rename,"
+#endif
+#ifdef SYS_renameat
+	  "renameat,"
+#endif
+#ifdef SYS_renameat2
+	  "renameat2,"
+#endif
+#ifdef SYS_rmdir
+	  "rmdir,"
+#endif
+#ifdef SYS_setxattr
+	  "setxattr,"
+#endif
+#ifdef SYS_stat
+	  "stat,"
+#endif
+#ifdef SYS_stat64
+	  "stat64,"
+#endif
+#ifdef SYS_statfs
+	  "statfs,"
+#endif
+#ifdef SYS_statfs64
+	  "statfs64,"
+#endif
+#ifdef SYS_statx
+	  "statx,"
+#endif
+#ifdef SYS_symlink
+	  "symlink,"
+#endif
+#ifdef SYS_symlinkat
+	  "symlinkat,"
+#endif
+#ifdef SYS_truncate
+	  "truncate,"
+#endif
+#ifdef SYS_truncate64
+	  "truncate64,"
+#endif
+#ifdef SYS_unlink
+	  "unlink,"
+#endif
+#ifdef SYS_unlinkat
+	  "unlinkat,"
+#endif
+#ifdef SYS_utime
+	  "utime,"
+#endif
+#ifdef SYS_utimensat
+	  "utimensat,"
+#endif
+#ifdef SYS_utimes
+	  "utimes"
+#endif
+	},
+	{ .name = "@io-event", .list =
+#ifdef SYS__newselect
+	  "_newselect,"
+#endif
+#ifdef SYS_epoll_create
+	  "epoll_create,"
+#endif
+#ifdef SYS_epoll_create1
+	  "epoll_create1,"
+#endif
+#ifdef SYS_epoll_ctl
+	  "epoll_ctl,"
+#endif
+#ifdef SYS_epoll_ctl_old
+	  "epoll_ctl_old,"
+#endif
+#ifdef SYS_epoll_pwait
+	  "epoll_pwait,"
+#endif
+#ifdef SYS_epoll_wait
+	  "epoll_wait,"
+#endif
+#ifdef SYS_epoll_wait_old
+	  "epoll_wait_old,"
+#endif
+#ifdef SYS_eventfd
+	  "eventfd,"
+#endif
+#ifdef SYS_eventfd2
+	  "eventfd2,"
+#endif
+#ifdef SYS_poll
+	  "poll,"
+#endif
+#ifdef SYS_ppoll
+	  "ppoll,"
+#endif
+#ifdef SYS_pselect6
+	  "pselect6,"
+#endif
+#ifdef SYS_select
+	  "select"
+#endif
+	},
+	{ .name = "@ipc", .list =
+#ifdef SYS_ipc
+	  "ipc,"
+#endif
+#ifdef SYS_memfd_create
+	  "memfd_create,"
```

Comment:
also blocked from `memory-deny-write-execute`

Reply:
Yes, but I don't think that's so much of a problem, especially with the exceptions in place.